processor/processor.go

```diff
@@ -1564,7 +1564,7 @@ func (proc *HandleT) Store(in storeMessage) {
 		proc.logger.Debug("[Processor] Total jobs written to batch router : ", len(batchDestJobs))
 
 		err := proc.batchRouterDB.WithStoreSafeTx(func(tx jobsdb.StoreSafeTx) error {
-			err := proc.batchRouterDB.StoreInTx(tx, batchDestJobs)
+			err := proc.batchRouterDB.StoreInTx(ctx, tx, batchDestJobs)
```

Comment:
in the next step, we need to specialize each context: Instead of using the `storeTimeout` context for all operations inside `Store`, each operation should have its own, shorter context, using `storeTimeout` as a parent context.